### PR TITLE
Use Alpine as base Docker image and bump dependencies

### PR DESCRIPTION
**Pull request by stianpr**
Should work like before and we should see an overall decrease in image size by 50%.

### CODE REVIEW

**Review by asteinlein — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+3.0.0-beta / 30-10-2019
+==================
+
+* Use Alpine as base docker image. This will reduce the overall size of the image
+* by over 50%.
```
Comment (fdanielsen) [inline CHANGES.md]: "by over 50%" shouldn't be its own bullet point ;)

File: package.json [hunk 2]
```diff
@@ -27,10 +27,9 @@
     "url": "https://github.com/eliksir/screenie-server/issues"
   },
   "dependencies": {
-    "koa": "1.4.1",
-    "puppeteer": "1.0.0",
+    "koa": "2.11.0",
+    "puppeteer": "2.00.0",
```
Comment (fdanielsen) [inline package.json]: Version should be "2.0.0".

File: Dockerfile [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
-      libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
-      libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
-      libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \
-      ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget && \
-    wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64.deb && \
-    dpkg -i dumb-init_*.deb
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Installs make, g++, python and Chromium (77) packages.
```
Comment (fdanielsen) [inline Dockerfile]: Can we align with Puppeteer 2.0.0 which comes with Chromium 79.0.3942.0 by default?

**Review by fdanielsen — CHANGES_REQUESTED** [review]
Some minor feedback, but would also be good to look into aligning Chromium version in Docker with the default version in Puppeteer 2.0.0.

File: Dockerfile (hunk 3 above)
Comment (stianpr) [reply Dockerfile]: No, latest alpine 3.10 has only Chromium 77 available, but it does work with Puppeteer 2.0. [Even alpine edge has only v77](https://pkgs.alpinelinux.org/package/edge/community/x86_64/chromium).

File: package.json (hunk 2 above)
Comment (stianpr) [reply package.json]: a true.

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+3.0.0-beta / 31-10-2019
+==================
+
+* Use Alpine as base docker image
```
Comment (fdanielsen) [inline CHANGES.md]: "Docker"

File: package.json [hunk 5]
```diff
@@ -17,7 +17,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 8.0.0"
```
Comment (fdanielsen) [inline package.json]: I think we just require node 7.6.0 now given the Koa update. It's puppeteer 2.0.0 which requires 8.0.0.

**Review by fdanielsen — APPROVED** [review]

**Pull request merged**